dds_web/templates/404.html

```diff
@@ -12,7 +12,7 @@
 
 <p>
     You can get back to the homepage <a href="{{ url_for('pages.home') }}">here</a>.
-    If in doubt, please get in touch with the SciLifeLab Data Centre.
+    If in doubt, please get in touch with {{ 'the SciLifeLab Data Centre (datacentre@scilifelab.se)' if g.current_user and g.role and g.role != 'Researcher' else 'support'}}.
```

Comment:
> ...If in doubt, please get in touch with support.

I think this will still be misleading for the "end" users. In their eyes this is the Data Centre's system/service and they will always try to reach us. 
On the other hand, since they can have projects from different Units, it is not clear, in general, which unit they should contact, and I don't really have a better suggestion.

Reply:
I agree. In that case we'd have to change it to 
```
If in doubt, please get in touch with {{ 'the SciLifeLab Data Centre (datacentre@scilifelab.se)' if g.current_user and g.role and g.role != 'Researcher' else 'one of the SciLifeLab platforms delivering the data in your projects'}}. 
```
or something like that